source/type/pb_converter.cpp

```diff
@@ -991,7 +987,10 @@ compute_disruption_status(const nd::Impact& impact,
 template <typename P>
 void PbCreator::Filler::fill_message(const boost::shared_ptr<nd::Impact>& impact,
                                      P pb_object){
-    *pb_object->add_impact_uris() = impact->uri;
+    using boost::algorithm::none_of;
```

Comment:
Minor: maybe a little comment, as the code is not as simple as the goal.

Reply:
Done